flexget/plugins/output/download.py

```diff
@@ -155,9 +155,9 @@ def save_error_page(self, entry, task, page):
         received = os.path.join(task.manager.config_base, 'received', task.name)
         if not os.path.isdir(received):
             os.makedirs(received)
-        filename = os.path.join(received, '%s.error' % entry['title'].encode(sys.getfilesystemencoding(), 'replace'))
+        filename = os.path.join(received, '%s.error' % entry['title'].encode(sys.getfilesystemencoding(), 'replace').decode(sys.getfilesystemencoding()))
```

Comment:
Hmm, we are encoding then decoding?? that does not sound right?

what error did you get?